.github/scripts/test_trymerge.py

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env python3
+from trymerge import GitHubPR
+from unittest import TestCase, main
+
+class TestGitHubPR(TestCase):
+    def test_get_author_null(self):
+        """ Tests that PR author can be computed
+            If reply contains NULL
+        """
+        pr = GitHubPR("pytorch", "pytorch", 71759)
+        author = pr.get_author()
+        self.assertTrue(author is not None)
+        self.assertTrue("@" in author)
+
+
+if __name__ == "__main__":
+    import os
+    token = os.environ.get("GITHUB_TOKEN")
+    if token is None:
+        print("Test requires valid github token")
+    else:
+        main()
```

Comment:
It might be better to mock the API response to reduce the dependency on GitHub API functionality, but this is fine as-is

Reply:
Do you know a good python library for mocking graphql requests?

Reply:
I would recommend just using the built-in python mocking to mock `gh_graphql`. Anything more complicated than that seems like overkill at this point

Reply:
Updated, looks a bit ugly, but that's what it is